analytics/pom.xml

```diff
@@ -1,599 +1,704 @@
 <?xml version="1.0"?>
```

Comment:
I did not expect such a reindentation on the pom, sorry about that. Maybe we shall stick with 2 spaces on xml files ?